e2e/test/iothub/CombinedClientOperationsPoolAmqpTests.cs

```diff
@@ -48,6 +48,7 @@ public async Task DeviceCombinedClientOperations_SingleConnection_AmqpWs()
                 .ConfigureAwait(false);
         }
 
+        [Ignore]
```

Comment:
You'll want to add a comment calling out why this test was ignored.

Reply:
added